code/modules/admin/verbs/debug.dm

```diff
@@ -359,6 +359,27 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
 		message_admins("[key_name_admin(src)] has deleted all instances of [hsbitem].", 0)
 	feedback_add_details("admin_verb","DELA") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_debug_del_sing()
+	set category = "Debug"
+	set name = "Del Singulo / Tesla"

```

Comment:
Hey, as it is I left out that it will delete all Nar'Sies, Tesla gets a mention.